### PR TITLE
Changes added to SRC Main.

### PR DESCRIPTION
**Pull request by jmadde882**
Enemies in red have been added.

UI text box has been added.

### CODE REVIEW

**Comment by borisv1307** [comment]
Please remove the .idea folder and all files. Those shouldn't go into git.

**Review by borisv1307 — CHANGES_REQUESTED** [review]

**Comment by jmadde882** [comment]
Okay I’ll adjust that tomorrow. Are you able to run the game and see the enemy movement?
________________________________

File: src/main/java/engine/GameEngine.java [hunk 1]
```diff
@@ -1,106 +1,249 @@
...
+
+    public boolean isExit() {
+        return exit;
+    }
+
+    public void setExit(boolean exit) {
+        this.exit = exit;
+    }
+
+    public boolean setEnemyCount(int count) {
```
[the remark this answers is not in the document]
Comment (jmadde882) [reply src/main/java/engine/GameEngine.java:148]: adressed

File: src/main/java/engine/GameEngine.java [hunk 2]
```diff
@@ -1,106 +1,249 @@
...
+        SecureRandom random = new SecureRandom();
+
+        int enemycount = 0;
+        for (int i = 0; i < count; i++) {
+            while (!created) {
+                x = (int) Math.round(random.nextDouble()) * width;
+                y = (int) Math.round(random.nextDouble()) * height;
+                type = tiles.get(new Point(x, y));
+
+                if (type == TileType.PASSABLE && !((x == 0 || x == 1) && (y == 0 || y == 1))) {
```
[the remark this answers is not in the document]
Comment (jmadde882) [reply src/main/java/engine/GameEngine.java]: fixed

File: src/main/java/engine/GameEngine.java [hunk 3]
```diff
@@ -1,106 +1,249 @@
...
+        int x;
+        int y;
+        TileType type;
+
+        enemies.clear();
+
+        SecureRandom random = new SecureRandom();
+
+        int enemycount = 0;
+        for (int i = 0; i < count; i++) {
```
[the remark this answers is not in the document]
Comment (jmadde882) [reply src/main/java/engine/GameEngine.java]: fixed

File: src/main/java/engine/GameEngine.java [hunk 4]
```diff
@@ -1,106 +1,249 @@
...
+                }
+            }
+
+            created = false;
+        }
+
+        return enemycount > 0;
+    }
+
+    public int randomPosition() {
```
[the remark this answers is not in the document]
Comment (jmadde882) [reply src/main/java/engine/GameEngine.java:198]: fixed

[the remark this answers is not in the document]
Comment (jmadde882) [reply src/main/java/engine/GameEngine.java]: fixed

**Pull request closed without merging**